crates/pallet-domains/src/staking.rs

```diff
@@ -711,10 +711,62 @@ pub(crate) fn do_deregister_operator<T: Config>(
     })
 }
 
+/// Different type of withdrawal
+///
+/// NOTE: the deposit happen in the current epoch may not able to withdraw
+// until the current epoch end
```

Comment:
This comment was unclear to me, is it about withdrawing deposits made during the current epoch?

```suggestion
/// NOTE: if the deposit was made in the current epoch, the user may not be able to withdraw it
// until the current epoch ends
```